scripts/test-harness.sh

```diff
@@ -174,6 +174,22 @@ while getopts "n:t:x:d:s:l:" opt; do
     l)  if valid_language_name "${OPTARG}"; then LANGUAGE=${OPTARG};
         else echo "Invalid language name ${OPTARG}" && help && exit 1;
         fi ;;
+    c)  if [[ ${OPTARG} -eq 1 ]]; then
+            if [[ "$LANGUAGE" == "c" ]]; then
```

Comment:
check for both "c" and "C"

Reply:
Yeap, C/C++ are the same. Add a new check condition.